org/org.yaml

```diff
@@ -153,6 +154,7 @@ orgs:
         maintainers:
         - vdemeester
         - dlorenc
+        - priyawadhwa
```

Comment:
NIT: I know this is a bit confusing: `members` of the `chains.maintainers` github team are `maintainers` of the `chains` projects. `maintainers of the `chains.maintainers` github team are allowed to manage the team - but that role does not really matter sync we manage the teams via this file and peribolos 😅 
Anyways, since maintainers of a team are also members, this works fine.

Note that we don't sync this to the OWNERS files in the repos (yet?) so you'll need an extra PR to https://github.com/tektoncd/chains/blob/main/OWNERS